auth-web/src/views/auth/staff/AdminDashboardView.vue

```diff
@@ -0,0 +1,256 @@
+              <v-col cols="12" sm="3" class="pr-4">
+                <strong>Business Name</strong>
+              </v-col>
+              <v-col cols="12" sm="9">
+                {{ businessDetails.legalName }}
+              </v-col>
+            </v-row>
+            <v-row no-gutters>
+              <v-col cols="12" sm="3" class="pr-4">
+                <strong>Registration Number</strong>
+              </v-col>
+              <v-col cols="12" sm="9">
+                {{ businessDetails.identifier }}
+              </v-col>
+            </v-row>
+            <v-row no-gutters>
+              <v-col cols="12" sm="3" class="pr-4">
+                <strong>Business Number (BN9/BN15)</strong>
+              </v-col>
+              <v-col cols="12" sm="9">
+                {{ businessDetails.taxId || '(Not Available)' }}
+              </v-col>
+            </v-row>
+
+            <v-expand-transition>
+              <div v-show="submitBNRequestErrorMessage">
+                <v-alert type="error" icon="mdi-alert-circle" class="mb-0"
+                  >{{ submitBNRequestErrorMessage }}
+                </v-alert>
+              </div>
+            </v-expand-transition>
+
+            <v-form
+              ref="submitBNRequestForm"
+              v-on:submit.prevent="submitBNRequest"
+              class="mt-3"
+            >
+              <v-checkbox
+                v-model="requestCRA"
+                class="mt-0 pt-0"
+                label="Request CRA to assign a Program Account (BN15)"
+                :disabled="!canRequestNewBN()"
+              >
+              </v-checkbox>
+              <!-- Business Number -->
+              <v-text-field
+                dense req
+                filled
+                persistent-hint
+                class="business-number"
+                label="Business Number (Optional)"
+                hint="First 9 digits of the CRA Business Number"
+                v-model="businessNumber"
+                v-mask="['#########']"
+                :rules="businessNumberRules"
+                :disabled="!requestCRA"
+              />
+              <v-btn
+                color="primary"
+                class="submit-btn mt-0"
+                type="submit"
+                depressed
+                :disabled="!isBNRequestFormValid()"
+                :loading="submitActive"
+              >
+                <span>Submit</span>
+              </v-btn>
+            </v-form>
+          </template>
+        </v-card>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script lang="ts">
+/* eslint-disable */
+import { Component } from 'vue-property-decorator'
+import CommonUtils from '@/util/common-util'
+import Vue from 'vue'
+import { mask } from 'vue-the-mask'
+import { mapActions } from 'vuex'
+import { BNRequest } from '@/models/request-tracker'
+
+
+@Component({
+  methods: {
+    ...mapActions('business', ['createBNRequest', 'searchBusiness'])
```

Comment:
https://github.com/bcgov/sbc-auth/blob/4feabfa18a9b9b110d244ab6376156f32c9c72f9/auth-web/src/components/auth/staff/account-management/StaffActiveAccountsTable.vue#L190

Might want to use this kind of binding.. works way better with IDE and typescript in general.